lib/dicon/executor.ex

```diff
@@ -35,6 +35,18 @@ defmodule Dicon.Executor do
   @callback copy(conn, source :: char_list, target :: char_list) :: :ok | {:error, binary}
 
   @doc """
+  Tails the given list of `patterns` continuously (`tail -f`), returning only in
```

Comment:
Actually, let's wait for the discussion from https://github.com/lexmag/dicon/pull/27#discussion_r107050923.